swap/src/protocol/bob/state.rs

```diff
@@ -595,6 +588,18 @@ impl State6 {
         Ok(())
     }
 
+    pub async fn wait_for_cancel_timelock_to_expire(
```

Comment:
This is unused according to my IDE - and should not be on `State6` because it is only used after the cancel timelock has expired.

Reply:
Good catch, let me check!